cloudbuild.meta.yaml

```diff
@@ -46,6 +46,7 @@ steps:
     declare -a apps=("hello-world" "guestbook")
     gcloud builds submit . --config cloudbuild.tmpl.yaml --substitutions _DIR=${dir},_LANG=${lang},_APP=${apps[0]}
     gcloud builds submit . --config cloudbuild.tmpl.yaml --substitutions _DIR=${dir},_LANG=${lang},_APP=${apps[1]}
+    gcloud builds submit . --config golang/go-cloud-run-hello-world
```

Comment:
Per discussion with Simon, we do not need to use the substitutions since we're using a unique cloudbuild.yaml per sample.